src/iuse.cpp

```diff
@@ -5059,9 +5060,16 @@ static bool heat_item( player &p )
     item &target = heat->is_food_container() ? heat->contents.front() : *heat;
     p.mod_moves( -300 );
     add_msg( _( "You heat up the food." ) );
-    target.item_tags.insert( "HOT" );
-    target.active = true;
-    target.item_counter = 600; // sets the hot food flag for 60 minutes
+    if( target.item_tags.count( "FROZEN" ) ) {
+        target.item_tags.erase( "FROZEN" );
+        target.item_tags.insert( "HOT" );
```

Comment:
I feel the process of consecutive defrosting and heating should be longer than simple heating.

Reply:
You mean regarding moves consumed?

Reply:
Yes. Regular heating consumes 300 moves, while defrosting+heating should consume more.